src/components/pages/AddItem.jsx

```diff
@@ -1,6 +1,175 @@
+            <form css={addItem}>
+                <div css={contentHeader}>
+                    <h3>상품 등록하기</h3>
+                    <button 
+                        type="submit" 
+                        css={headerButton} 
+                        onClick={handleSubmitButton}
+                        disabled={
+                                itemName.trim() === "" || 
+                                itemDescription.trim() === "" ||
+                                itemPrice.trim() === "" ||
+                                itemHashTag.length === 0  ||
+                                !imgPreviewUrl
+                        }
+                    >
+                            등록
+                        </button>
+                </div>
+                <div css={itemWrapper}>
+                    <div >
+                        <p>상품 이미지</p>
+                        <div css={imgRowWrapper}>
+                        <label htmlFor='fileUpload' css={addItemImageWrapper}>
+                            <div css={addItemBox}>
+                                <img src={plusIcon} alt='추가' css={plusIconStyle} />
+                                <span>이미지 등록</span>
+                            </div>
+                        </label>
+                        {imgPreviewUrl && (
+                            <div css={previewBox}>
+                                <img src={imgPreviewUrl} alt='선택한 사진' css={previewImg} />
+                                <button onClick={handleFileDelete}>
+                                    <img src={deleteIcon} alt='삭제버튼' css={deleteImage} />
+                                </button>
+                            </div>
+                        )}
+                        </div>
+                        <input  
+                            id='fileUpload'
+                            type='file'
+                            accept='image/*'
+                            style={{display: 'none'}}
+                            onChange={handleFileChange}
+                        />    
+                    </div>
+
+                    {showWarning && (
+                        <p css={alertMessage}>이미지는 한 장만 선택할 수 있습니다.</p>
+                    )}
+
+                    <div>
+                        <label htmlFor='name'>상품명</label>
+                        <input id="name" placeholder="상품명을 입력해주세요" value={itemName} onChange={(e)=> setItemName(e.target.value)}></input>
+                    </div>
+                    
+                    <div>
+                        <label>상품 소개</label>
+                        <textarea id="description" placeholder="상품 소개를 입력해주세요" value={itemDescription} onChange={(e) => setItemDiscription(e.target.value)}></textarea>
+                    </div>
+
+                    <div>
+                        <label>판매가격</label>
+                        <input id="price" placeholder="판매가격을 입력해주세요" type='number' value={itemPrice} onChange={(e) => setItemPrice(e.target.value)}></input>
```

Comment:
💊 제안
라벨과 인풋을 사용하셨으니 연결해서 라벨 클릭시 자동으로 인풋으로 포커스가 가게 해주세요.

```suggestion
                        <label htmlFor="price">판매가격</label>
                        <input id="price" placeholder="판매가격을 입력해주세요" type='number' value={itemPrice} onChange={(e) => setItemPrice(e.target.value)}></input>
```